crates/ruff_python_formatter/resources/test/fixtures/ruff/statement/match.py

```diff
@@ -141,3 +141,24 @@ def foo():
     no_comments
     ):
         pass
+
+match foo:
+    case "a", "b":
+        pass
+    case ("a", "b"):
+        pass
+    case ["a", "b"]:
+        pass
```

Comment:
Is there a way to detect whether the sequence is a list or a tuple from?

Reply:
As it seems, not by looking at the AST. But you can get this information when inspecting the source:

```rust
let parentheses = match &f.context().source()[item.range()] {
	['(', ...] => Some(('(', ')')),
	['[', ...] => Some(('[', ']')),
	_ => None
```